sentry_sdk/transport.py

```diff
@@ -1,12 +1,16 @@
+from __future__ import print_function
```

Comment:
We don't need this on the 2.0 branch because there's no Python 2 support anymore

Reply:
Good catch – I think Git somehow put this line in when I was splitting the `Transport` `ABC` stuff into a separate PR